exercises/palindrome/test/test_palindrome.c

```diff
@@ -0,0 +1,50 @@
+#include "vendor/unity.h"
+#include "../src/palindrome.h"
+
+// isPalindrome tests
+void test_palindrome_even(void)
+{
+   TEST_ASSERT_EQUAL_INT(0, isPalindrome(91, 99));
+}
+
+void test_palindrome_even_fail(void)
+{
+   TEST_ASSERT_EQUAL_INT(1, isPalindrome(5, 7));
+}
+
+void test_palindrome_odd(void)
+{
+   TEST_ASSERT_EQUAL_INT(0, isPalindrome(11, 11));
+}
+
+void test_palindrome_odd_fail(void)
+{
+   TEST_ASSERT_EQUAL_INT(1, isPalindrome(10, 12));
+}
+
+// palindromeFactors tests
+void test_palindromes_list(void)
+{
+
+   PalindromeProduct *palind = getPalindromeProduct(1, 9);
+
+   TEST_ASSERT_EQUAL_INT(1, palind[0].palindrome);
+   TEST_ASSERT_EQUAL_INT(1, palind[0].factors[0].a);
+   TEST_ASSERT_EQUAL_INT(1, palind[0].factors[0].b);
+
+   TEST_ASSERT_EQUAL_INT(9, palind[1].palindrome);
+   TEST_ASSERT_EQUAL_INT(1, palind[1].factors[0].a);
+   TEST_ASSERT_EQUAL_INT(9, palind[1].factors[0].b);
+}
```

Comment:
This is the only test for `getPalindromeProduct` and I think we need significantly more testing of this function. For example, this doesn't show that the result can contain multiple factors for each palindrome.